docs/_index.md

```diff
@@ -67,10 +67,10 @@ poetry export --only test,docs
 
 * `--format (-f)`: The format to export to (default: `requirements.txt`). Currently, only `constraints.txt` and `requirements.txt` are supported.
 * `--output (-o)`: The name of the output file.  If omitted, print to standard output.
-* `--without`: The dependency groups to ignore when exporting.
-* `--with`: The optional dependency groups to include when exporting.
-* `--only`: The only dependency groups to include when exporting.
-* `--default`: Only export the main dependencies. (**Deprecated**)
+* `--with`: The optional and non-optional dependency groups to include. By default, only the main dependencies are included.
+* `--only`: The only dependency groups to include.
+* {{< option name="without" deprecated=true >}}The dependency groups to ignore.{{< /option >}}
+* {{< option name="default" deprecated=true >}}Only export the main dependencies.{{< /option >}}
 * {{< option name="dev" deprecated=true >}}Include development dependencies.{{< /option >}}
```

Comment:
I'm not sure if/where this syntax works. In the end, it's https://github.com/python-poetry/poetry/blob/88096aca2300fee8d1d0babdb505e1cf50cbea17/docs/cli.md?plain=1#L706-L737 what will be displayed on the website anyway. Probably, we should choose the simpler syntax (without `{{< option`).

Reply:
Is this page published at all? I found it weird to have to edit the same information at 2 places. I couldn't find the first sentence of this file through a google search:

> The export plugin allows the export of locked packages to various formats.

Should these files simply be deleted?

Reply:
Afaik, it's not published. However, I suppose it was the target to integrate it somehow in the poetry docs and remove the the command from https://github.com/python-poetry/poetry/blob/master/docs/cli.md. Thus, deleting it would be a step backwards. On the other hand, python-poetry/poetry#5980 got stuck since https://github.com/python-poetry/poetry/pull/5980#issuecomment-1231972150.

I just noticed, there is even a third place: https://github.com/python-poetry/poetry-plugin-export/blob/main/README.md

Since I'm not much into this topic, I'd probably just update all three spots. 🤷

Reply:
I couldn't find the third spot, but I edited the text a bit to make it consistent.